src/activities/discussions/DiscussionTopicEntity.js

```diff
@@ -6,6 +6,17 @@ import { performSirenActions } from '../../es6/SirenAction.js';
  * DiscussionTopicEntity class representation of a D2L Discussion Topic.
  */
 export class DiscussionTopicEntity extends Entity {
+	/**
+	 * @returns {string} the href of the discussion topic's discussion forum entity
+	 */
+	forumHref() {
+		if (this._entity) {
+			const href = this._entity.getLinkByRel(Rels.Discussions.forum).href;
```

Comment:
Might want to do a `this._entity.hasLinkByRel(Rels.Discussions.forum)` check here before getting the link

Reply:
Ah yes, good shout